### DISCUSSION TITLE
Solar Edge Grid Monitoring

### DISCUSSION THREAD

**Discussion by richardcastle** [WillCodeForCats/solaredge-modbus-multi#692] [Q&A]
I've had a grid issue that has recently sent my inverter into a "Grid Monitoring" cycle which causes the system to not produce. This status is quite clearly displayed within the local wifi network commissioning tool on my phone, however similar status isn't surfaced within this wonderful tool. The standard status field simply cycled between standby and production. My question is whether there are more fields that we can capture on the modbus, or is the current implementation an exhaustive list of what Solar Edge throw out onto the modbus for diagnostics? I'd love to be able to trigger an alert whenever the invertor got stuck in Grid Monitoring again! Thanks.

**Comment by WillCodeForCats** [comment]
The integration already presents all known modbus registers. There's only Status and Status Vendor values. You can check if Status Vendor shows something other than 0 consistently.

**Reply by richardcastle** [reply]
Thanks for clarifying. Looking back at the history, the Status Vendor was flicking between 0 and 134. I've had a look, but can't see any docs describing the vendor codes, unless anyone has anything?

As an aside, it turned out that the issue was caused by an isolation issue on the panels string - connector had properly failed and was waterlogged. So we were seeing the system trying to restart, but failing because of leakage to ground on the string.
